woocommerce-gateway-ebanx/includes/class-wc-ebanx-one-click.php

```diff
@@ -505,6 +484,93 @@ public function print_button() {
 
 		wc_get_template( 'one-click.php', $args, '', WC_EBANX::get_templates_path() . 'one-click/' );
 	}
+
+	/**
+	 * Calculates the max instalments allowed based on price, country and minimal instalment value
+	 * given by the credit-card acquirer
+	 *
+	 * @param  $price double Product price used as base
+	 * @return integer
+	 */
+	private function fetch_acquirer_max_installments_for_price($price) {
+		$max_instalments = WC_Ebanx_Gateway_Utils::MAX_INSTALMENTS;
+
+		switch (trim(strtolower(WC()->customer->get_country()))) {
```

Comment:
We have a method called `getTransactionAddress('country')`, maybe in the future we need to replace `trim(strtolower(WC()->customer->get_country()))` to this method.